js/components/wysiwyg.js

```diff
@@ -122,6 +122,10 @@ Fliplet.FormBuilder.field('wysiwyg', {
     }
   },
   mounted: function () {
+    if (this.source !== 'defaultSource') {
```

Comment:
couldn't this overly repeated logic be implemented in DRY fashion? I'm surprised we didn't manage to optimise this.

As a further insight, I think we would not have allowed this repetition on any other part of the system because we would have questioned it. cc @tonytlwu

Reply:
@squallstar We could move this logic to the separate function in the core.js file and in this way it would be common to all components and we can call it in components that we are needed.
Will it be good to do that way?

Reply:
@YaroslavOvdii what if the mounted function is defined by the system and the component's is called afterwards? This has been done for many other occasions in the form:

```
var mounted = component.mounted;
component.mounted = function () {
  // do something
  mounted.call(this);
}
```

Reply:
@squallstar at the moment we didn't use defined by the system mounted function because we were needed that our new feature work right after the component was loaded, but system mount function was added only after the component appears on the screen and it doesn't call mount again to use it. That is why in all components that must have this feature we have added mount function.

Reply:
@YaroslavOvdii I don't understand your last comment, but more importantly please just look at code that we wrote in the past if you need to see how we have created a default mounted method in the basic library https://github.com/Fliplet/fliplet-widget-form-builder/blob/master/js/libs/core.js#L272